gsql/cache/cache_key.py

```diff
@@ -0,0 +1,74 @@
+from functools import lru_cache
+from key_pair import Keypair
+from gsql.cache.cache_handler import (
+    CacheDict,
+    check_json_file_exists,
+    count_lines_in_json,
+    create_json_file,
+    keys_exists,
+    read_from_json,
+    remove_key_from_json,
+    write_to_json,
+)
```

Comment:
All this imports could be avoided by encapsulating them in a class

Reply:
yes, done